addons/website/static/src/snippets/s_table_of_content/000.js

```diff
@@ -38,7 +40,7 @@ const TableOfContent = publicWidget.Widget.extend({
      * @private
      */
     _updateTableOfContentNavbarPosition() {
-        if (!this.$target[0].querySelector('a.table_of_content_link')) {
+        if (!this.$el[0].querySelector('a.table_of_content_link')) {
```

Comment:
I took the opportunity to make this small change

Reply:
I'll change to `this.el` then